trunk/SUAVE/Methods/Constraint_Analysis/compressibility_drag_constraint.py

```diff
@@ -0,0 +1,78 @@
+## @ingroup Methods-Constraint_Analysis
+# Oswald_efficiency.py
+# 
+# Created:  Nov 2021, S. Karpuk
+# Modified: 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# SUAVE Imports
+import SUAVE
+import numpy as np
+
+# ------------------------------------------------------------------------------------
+#  Compute maximum lift coefficient for the constraint analysis
+# ------------------------------------------------------------------------------------
+
+## @ingroup Methods-Constraint_Analysis
+def compressibility_drag_constraint(mach,cl,geometry):
```

Comment:
This is a carbon copy of an existing script: https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py

Remove this script and reference the existing one